crates/re_space_view_tensor/src/space_view_class.rs

```diff
@@ -238,40 +206,25 @@ impl SpaceViewClass for TensorSpaceView {
 
         let tensors = &system_output.view_systems.get::<TensorSystem>()?.tensors;
 
-        if tensors.is_empty() {
-            ui.centered_and_justified(|ui| ui.label("(empty)"));
-            state.selected_tensor = None;
-        } else {
-            if let Some(selected_tensor) = &state.selected_tensor {
-                if !tensors.contains_key(selected_tensor) {
-                    state.selected_tensor = None;
-                }
-            }
-            if state.selected_tensor.is_none() {
-                state.selected_tensor = Some(tensors.iter().next().unwrap().0.clone());
-            }
-
-            if tensors.len() > 1 {
-                // Show radio buttons for the different tensors we have in this view - better than nothing!
-                ui.horizontal(|ui| {
-                    for instance_path in tensors.keys() {
-                        let is_selected = state.selected_tensor.as_ref() == Some(instance_path);
-                        if ui.radio(is_selected, instance_path.to_string()).clicked() {
-                            state.selected_tensor = Some(instance_path.clone());
-                        }
-                    }
-                });
-            }
+        if tensors.len() > 1 {
+            state.tensor = None;
 
-            if let Some(selected_tensor) = &state.selected_tensor {
-                if let Some((tensor_data_row_id, tensor)) = tensors.get(selected_tensor) {
-                    let state_tensor = state
-                        .state_tensors
-                        .entry(selected_tensor.clone())
-                        .or_insert_with(|| PerTensorState::create(*tensor_data_row_id, tensor));
-                    view_tensor(ctx, ui, state_tensor, *tensor_data_row_id, tensor);
-                }
+            egui::Frame {
+                inner_margin: re_ui::ReUi::view_padding().into(),
+                ..egui::Frame::default()
             }
+            .show(ui, |ui| {
+                ui.label(format!(
+                    "Can only show one tensor at a time; was given {}.",
```

Comment:
Error messages should be actionable!

That is: tell the user how to fix this ("Select the space view [click here] and blah blah…")